README.md

```diff
@@ -567,6 +567,105 @@ unsubscribe();
 
 You can remove all listeners at once by calling `reactions.unsubscribeAll()`.
 
+## Handling encoded objects in integrations
+
+If you have set up an [Ably integration](https://ably.com/docs/general/integrations) to receive events from your chat
+room, depending on your configuration, you may receive these as encoded objects.
+See [here](https://ably.com/docs/general/webhooks) for more information.
+Should you wish to convert this object to a chat type, you can use the functions below to help you.
+
+For example, if you have the following item reach your integration:
+
+```json
+{
+  "items": [
+    {
+      "webhookId": "Ja-tsg",
+      "source": "channel.message",
+      "serial": "108iZpUxQBe4Vv35120919@1720954404104-0",
+      "timestamp": 1720954404104,
+      "name": "channel.message",
+      "data": {
+        "channelId": "some-room::$chat::$chatMessages",
+        "site": "eu-west-1-A",
+        "messages": [
+          {
+            "id": "chat:6TP2sA:some-room:a4534b0ab37bdd5:0",
+            "clientId": "user1",
+            "timestamp": 1720954404104,
+            "encoding": "json",
+            "extras": {
+              "timeserial": "108iZpUxQBe4Vv35120919@1720954404104-0",
+              "headers": {}
+            },
+            "data": "{\"text\":\"some text data\",\"metadata\":{}}",
+            "name": "message.created"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+If you are unsure as to the underlying chat entity type of the encoded message,
+you can use `getEntityTypeFromEncoded` to retrieve it,
+then call either `chatMessageFromEncoded` or `reactionFromEncoded` depending on the entity type.
+
+```ts
+import { getEntityTypeFromEncoded, chatMessageFromEncoded, reactionFromEncoded, ChatEntityType } from '@ably/chat';
+
+integrationMessage.items.forEach((item) => {
+  item.data.messages.forEach(async (encodedMessage) => {
+    const entityType = getEntityTypeFromEncoded(encodedMessage);
+    switch (entityType) {
+      case ChatEntityType.ChatMessage:
+        const chatMessage = await chatMessageFromEncoded(encodedMessage);
+        console.log(chatMessage);
+        break;
+      case ChatEntityType.Reaction:
+        const reaction = await reactionFromEncoded(encodedMessage);
+        console.log(reaction);
+        break;
+      default:
+        console.log('Unknown entity type');
+    }
+  });
+});
+```
+
+## Handling ably inbound messages
+
+It may be the case that you wish to convert an Ably inboundMessage to a chat type, such as if you are working with the
```

Comment:
We don't really document `InboundMessage` anywhere as a thing - it's niche to ably-js. Perhaps something like "If you're working with Channels directly and not using the Chat SDK?"

Reply:
I can change the blurb here and the title above, but considering the types are inbound messages, it feels a bit strange to obscure that fact here in the readme?

Reply:
Ahh I see, yes in that case I'll update this to your phrasing there.